gradle.properties

```diff
@@ -1,3 +1,7 @@
 version=1.0-SNAPSHOT
 android.useAndroidX=true
-android.enableJetifier=true
\ No newline at end of file
+android.enableJetifier=true
```

Comment:
why it still use `android.enableJetifier=true` ? as I saw in allure-kotlin-android you don't use any old support libraries

Reply:
Yup, seems redundant ATM. When this code was being written some libraries were still not ported to androidx.